client/client.go

```diff
@@ -94,16 +94,24 @@ func ValidRedirectURL(rURL *url.URL, redirectURLs []url.URL) (url.URL, error) {
 	return url.URL{}, ErrorInvalidRedirectURL
 }
 
-func ClientsFromReader(r io.Reader) ([]Client, error) {
+// LoadableClient contains sufficient information for creating a Client and its related entities.
+type LoadableClient struct {
```

Comment:
I'm kinda iffy about this as a top level object. It shouldn't really be used outside of ClientsFromReader (which is only used in server/config.go).